functions/lib/blockiesPNG.js

```
@@ -79,12 +84,29 @@ function fillRect(png, x, y, w, h, color) {
   }
 }
 
-function render(opts) {
+async function render(opts) {
   opts = buildOptions(opts);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)